src/main/webapp/app/exercises/programming/manage/grading/tasks/programming-exercise-task.service.ts

```diff
@@ -258,7 +258,6 @@ export class ProgrammingExerciseTaskService {
 /**
  * Gets a single value from a list if there is only one unique value. otherwise returns undefined
  * @param values
- * @private
  */
 const getSingleValue = (values: any[]) => {
```

Comment:
There is no `private` keyword here because file-level constants are visible within the file only anyway and this is a const.